config.json

```diff
@@ -1832,11 +1846,21 @@
       "slug": "range-iteration",
       "uuid": "8f644992-59ec-4fff-8b40-36b5ed61404d"
     },
+    {
+      "name": "Runes",
+      "slug": "runes",
+      "uuid": "6b221481-bbb3-4ccf-8be1-a5e0a20bbd01"
```

Comment:
I had to add this as `logs-logs-logs` no longer has `strings-package` in its concepts and needed at least one concept.